indico/modules/categories/client/js/index.js

```diff
@@ -5,64 +5,31 @@
 // modify it under the terms of the MIT License; see the
 // LICENSE file for more details.
 
-import './display';
+import categoryStatisticsURL from 'indico-url:categories.statistics';
+
+import React from 'react';
+import ReactDOM from 'react-dom';
 
-(function(global) {
-  global.setupCategoryStats = function setupCategoryStats() {
-    $(document).ready(() => {
-      $('.plot-container .plot').each((__, element) => {
-        const $this = $(element);
-        const xMin = $this.data('min-x');
-        const xMax = $this.data('max-x');
-        const yMin = $this.data('min-y');
-        const yMax = $this.data('max-y');
+import {indicoAxios, handleAxiosError} from 'indico/utils/axios';
+
+import './display';
+import CategoryStatistics from './components/CategoryStatistics';
 
-        var data = $this.data('values') || {};
+document.addEventListener('DOMContentLoaded', async () => {
```

Comment:
I'd keep this wrapped in a `setupCategoryStats` function to avoid running it on every category page where you don't use stats at all